src/gsi/read_dbz_nc.f90

```diff
@@ -337,6 +338,11 @@ subroutine read_dbz_nc(nread,ndata,nodata,infile,lunout,obstype,sis,hgtl_full,no
 
   call w3fs21(iadate,mins_an)  !mins_an -integer number of mins snce 01/01/1978
   rmins_an=mins_an             !convert to real number
+  if(l4dvar.or.l4densvar) then
+     timeb=real(mins_an-iwinbgn,r_kind)
+  else
+     timeb=zero
```

Comment:
Why could we just use the " timeb=real(mins_an-iwinbgn,r_kind)"  ?  The obs relative to the analysis time could be non-zero even for 3dvar .  When the fget is used for 3dvar, the non-zero timeb would also produce differences for timeb=0 case.  So, I think if the general use of "timeb=real(mins_an-iwinbgn,r_kind)"  does work,  this should be unanimously used.

Reply:
In my understanding, the radar reflectivity NetCDF file doesn't include the observation time and it is assumed that the observation time is the same as the analysis time. That is why I used "timeb=real(mins_an-iwinbgn,r_kind)" for 4DVar and 4DEnVar.

> When the fget is used for 3dvar, the non-zero timeb would also produce differences for timeb=0 case. 

Are you pointing out the case of FGAT (First Guess at Appropriate Time)? I forgot to think about FGAT. I will fix to use "timeb=real(mins_an-iwinbgn,r_kind)" also in the 3DVar case.